frontend/src/app/modules/main/main-menu/main-menu.component.ts

```diff
@@ -183,6 +189,34 @@ export class MainMenuComponent implements OnDestroy, OnInit {
     }
   }
 
+  makeSecret() {
+    this._tempPlaylist.accessType = AccessType.secret;
+    this._playlistsService.changeAccessType(this._tempPlaylist)
+      .pipe(takeUntil(this._unsubscribe$))
+      .subscribe({
+        next: (data) => {
+          const playlistIndex = this.playlists.findIndex((pl) => pl.id === this._tempPlaylist?.id);
+          this.playlists[playlistIndex].accessType = data.accessType;
+          this.editedPlaylist = {} as PlaylistName;
+          this._tempPlaylist = {} as PlaylistName;
+        }
+      });
+  }
+
+  makeDefault() {
+    this._tempPlaylist.accessType = AccessType.default;
+    this._playlistsService.changeAccessType(this._tempPlaylist)
+      .pipe(takeUntil(this._unsubscribe$))
+      .subscribe({
+        next: (data) => {
+          const playlistIndex = this.playlists.findIndex((pl) => pl.id === this._tempPlaylist?.id);
+          this.playlists[playlistIndex].accessType = data.accessType;
+          this.editedPlaylist = {} as PlaylistName;
+          this._tempPlaylist = {} as PlaylistName;
+        }
+      });
+  }
+
```

Comment:
You can make makeSecret() and makeDefault() into one function that takes AccessType as an argument